content/patterns/multicloud-gitops-sgx-hello-world/mcg-sgx-hello-world-getting-started.adoc

```diff
@@ -0,0 +1,325 @@
++
+.List of Installed Operators for Multicloud GitOps Validated Pattern with SGX
+image::multicloud-gitops-sgx-hello-world/sgx-installed-operators.png[Multicloud GitOps Hub with SGX installed operators,role="related thumb right"]
++
+
+
+== Deploying the cluster by using the Validated Patterns Operator
+
+To install the *Validated Patterns Operator*:
+
+. Log in to the *Openshift Container Platform* web console and select *Operators > OperatorHub*.
+
+. Search for *Validated Patterns Operator*, open it and click *Install*.
++
+.Install Validated Patterns Operator
+image::multicloud-gitops-sgx-hello-world/sgx-validated-pattern-operator.png[Install Validated Patterns Operator,scale=50]
++
+. Choose default settings for the installation mode, namespaces and update strategy and confirm it by clicking *Install*.
+
+. Select *Operators > Installed Operators*.
+
+. Ensure that *Validated Patterns Operator* is listed in the `openshift-operators` project with a status `Succeeded`.
+
+After succeeded installation open *Validated Patterns Operator*, go to *Pattern* tab and click *Create Pattern*.
+
+. Fill the *Name* of the pattern `multicloud-gitops-sgx-hello-world` and *Cluster Group Name* `hub` (from _values-global.yaml_ file).
+
+. Under *Git Config > Target Repo* copy the link to your fork and under *Git Config > Target Revision* write the name of your branch (Figure 3).
+
+. Click *Create* to create the pattern.
++
+.Create Pattern Form
+image::multicloud-gitops-sgx-hello-world/sgx-create-pattern.png[Create Pattern Form]
++
+
+
+Verify that the rest of Operators have been installed:
+
+. To verify, in the *OpenShift Container Platform* web console, navigate to *Operators → Installed Operators* page.
+. Check that the following Operators are installed with  `Succeeded` status (Figure 1): 
+* *Advanced Cluster Management for Kubernetes* 
+* *Intel Device Plugins Operator* 
+* *multicluster engine for Kubernetes* 
+* *Node Feature Discovery Operator* 
+* *Red Hat Openshift GitOps*
+
+Add a secret for `config-demo` application (from _values-secret-multicloud-gitops.yaml_) to *Vault* manually:
+
+. Go to Vault service route. URL can be found:
+
+.. by running command:
++
+[source,terminal]
+----
+oc -n vault get route vault -ojsonpath='{.spec.host}'
+----
++
+.. in *Openshift Container Platform* web console under *Networking > Routes* for `vault` project.
+
+. Log into the Vault using root token. Root token can be found by executing command:
++
+[source,terminal]
+----
+oc -n imperative get secrets vaultkeys -ojsonpath='{.data.vault_data_json}' | base64 -d
+----
++
+
+. After login go to `secret` catalog and clik *Create secret* and fill all the fields manually (Figure 2):
+
+.. *Path for this secret* is `global/config-demo` (from _values.yaml_ file for `config-demo` charts)
+
+.. Under *Secret data* key is `secret` (from _values-secret-multicloud-gitops.yaml_ file) and in next field put its value.
+
+.. Click *Add* and then *Save*.
```

Comment:
TBH I don't get it... Let me try to rephrase it, and decide if this is clear:

a) Put `global/config-demo` value in the _Path for this secret_ field (the value cames from _values-secret-multicloud-gitops.yaml_ file).
b) Add one _Secret data_ key-value pair. Put `secret` as a key (left field). The value must acquired from _values-secret-multicloud-gitops.yaml_ file. Click _Add_ button to confirm.
c) Click _Save_ to save changes.

THIS IS HOW I UNDERSTAND THIS PART - PLEASE VALIDATE A LOGIC BEFORE REPLACING YOUR PART ;)

**BTW is this section twice? In the "Getting started" page, and in the "Demo Script"?**

Reply:
Checked and corrected. Left this part only in "getting started" section.